src/Evolution/DgSubcell/TwoMeshRdmpTci.hpp

```diff
@@ -0,0 +1,74 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <algorithm>
+
+#include "DataStructures/Variables.hpp"
+#include "Evolution/DgSubcell/Tags/Inactive.hpp"
+#include "Utilities/ErrorHandling/Assert.hpp"
+#include "Utilities/TMPL.hpp"
+
+namespace evolution::dg::subcell {
+/*!
+ * \brief Troubled cell indicator using a relaxed discrete maximum principle,
+ * comparing the solution on two grids at the same point in time.
+ *
+ * Checks that the subcell solution \f$\underline{u}\f$ and the DG solution
+ * \f$u\f$ satisfy
+ *
+ * \f{align*}{
+ * \min(u)-\delta \le \underline{u} \le \max(u)+\delta
+ * \f}
+ *
+ * where
+ *
+ * \f{align*}{
+ * \delta = \max\left[\delta_0, \epsilon(\max(u) - \min(u))\right]
+ * \f}
+ */
+template <typename... EvolvedVarsTags>
+bool two_mesh_rdmp_tci(
+    const Variables<tmpl::list<EvolvedVarsTags...>>& dg_evolved_vars,
+    const Variables<tmpl::list<Tags::Inactive<EvolvedVarsTags>...>>&
+        subcell_evolved_vars,
+    const double rdmp_delta0, const double rdmp_epsilon) noexcept {
```

Comment:
Would it make sense to add ASSERTS (or other kind of check) that these two constants are positive? I'm guessing the TCI wants these positive...